src/renderer/account/components/TransactionsPanel/TransactionsPanel.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+import { string, objectOf } from 'prop-types';
+
+import Panel from 'shared/components/Panel';
+import Tabs from 'shared/components/Tabs';
+import SendIcon from 'shared/images/account/send.svg';
+import ReceiveIcon from 'shared/images/account/receive.svg';
+
+import Send from './Send';
+import Receive from './Receive';
+import IconTab from './IconTab';
+import balanceShape from '../../shapes/balanceShape';
+import styles from './TransactionsPanel.scss';
+
+const TAB_SEND = 'send';
+const TAB_RECEIVE = 'receive';
+
+const TABS = {
+  [TAB_SEND]: <IconTab renderIcon={SendIcon}>Send</IconTab>,
+  [TAB_RECEIVE]: <IconTab renderIcon={ReceiveIcon}>Receive</IconTab>
+};
+
+export default class TransactionsPanel extends React.PureComponent {
+  static propTypes = {
+    className: string,
+    balances: objectOf(balanceShape).isRequired
+  };
+
+  static defaultProps = {
+    className: null
+  };
+
+  state = {
+    tab: TAB_SEND
+  };
+
+  render() {
+    return (
+      <Panel className={classNames(styles.transactionsPanel, this.props.className)}>
+        <Tabs
+          tabsClass={styles.tabs}
+          tabClass={styles.tab}
+          tabs={TABS}
+          selectedTab={this.state.tab}
```

Comment:
Just a question - I see that we almost never destructure, is this a rule we disabled? I thought it was a "required" thing to do with the latest version of eslint. I personally have had some benefits with always destructuring.

This should not be addressed right now though :P Just wondering about the fact and if we should/should not

Reply:
Are you talking about having to do this?

```js
const { tab } = this.state;
return <Tabs selectedTab={tab} />;
```

Reply:
Yes, at work I am forced to do so everywhere :P